tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist = tests
 skipsdist = true
-minversion = 3.25.0
```

Comment:
`minversion` is no longer needed because the new `tox>=3.25.0,<4` line below covers it